src/client/lobby/room-creator.js

```diff
@@ -4,34 +4,39 @@ import config from '../config/config';
 
 import style from './lobby.scss';
 
-export default class RoomCreator extends Component {
-
-    onUsernameClick(){
-        var username = document.getElementById('user-name').value;
-        this.props.setUsername(username);
-    }
+var username;
```

Comment:
This will make testing username and roomname have been set extremely difficult won't it? (see comment in componentDidMount)